security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -85,7 +85,7 @@ The following advisory is available for the {cert-manager-operator} 1.10.2:
 
 * link:https://access.redhat.com/errata/RHEA-2023:1238[RHEA-2023:1238]
 
-For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+The `1.10.2` version of the {cert-manager-operator} is based on the upstream cert-manager version `v1.10.2`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10/#v1102-changes-since-v1101[cert-manager project release notes for v1.10.2].
```

Comment:
Nit:

```suggestion
Version `1.10.2` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.10.2`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10/#v1102-changes-since-v1101[cert-manager project release notes for v1.10.2].
```